### DISCUSSION TITLE
Override default browser

### DISCUSSION THREAD

**Discussion by emagieika** [wezterm/wezterm#6162] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### Which Wayland compositor or X11 Window manager(s) are you using?

N/A

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

Hello,
Is there a way to override the default browser used by wezterm with something else? 

The reason I'm asking is that I use 2 browsers, usually Firefox, but for looking up documentation and other stuff often launched from the terminal - qutebrowser.

**Comment by wez** [comment, marked as answer]
https://wezfurlong.org/wezterm/config/lua/window-events/open-uri.html

**Reply by emagieika** [reply]
Thank you!
